js/src/field-attachment.js

```diff
@@ -1,152 +0,0 @@
-var sui = require('sui-utils/sui'),
-    editAttributeField = require( 'sui-views/edit-attribute-field' ),
-    $ = require('jquery');
-
-// Cache attachment IDs for quicker loading.
-var iDCache = {};
-
-sui.views.editAttributeFieldAttachment = editAttributeField.extend( {
```

Comment:
Nothing. I'm not sure why Github is showing it as a removal, it's really just being moved to `js/src/views/edit-attribute-field-attachment.js` and built into the `shortcode-ui.js` bundle.